config/config.go

```diff
@@ -526,6 +528,24 @@ func ConfigureDirectories() error {
 		return err
 	}
 
+	log.WithField("filepath", "/etc/pterodactyl/passwd").Debug("ensuring passwd file exists")
+	if passwd, err := os.Create("/etc/pterodactyl/passwd"); err != nil {
```

Comment:
The path used here should be configurable, and an enable option gating this feature should be added as well.

Reply:
I think this should be enabled by default at least for all servers as it resolves the "user not found" and "I have no name" errors that we see regularly.

Reply:
> I think this should be enabled by default at least for all servers as it resolves the "user not found" and "I have no name" errors that we see regularly.

It can be enabled by default, but there needs to be a way to disable it.